css-canvas-medical/styles.css

```diff
@@ -0,0 +1,245 @@
+body {
+  font-family: Avenir, sans-serif;
+  color: #6e6e6e;
+}
+
+.flex-row {
+  display: flex;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.justify-end {
+  justify-content: flex-end;
+}
+
+.justify-around {
+  justify-content: space-around;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.column-one-half {
+  flex-basis: 50%;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fifth {
+  flex-basis: 20%;
+}
+
+.column-two-fifths {
+  flex-basis: 40%;
+}
+
+.column-three-fifths {
+  flex-basis: 60%;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.width-90 {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+header {
+  height: 55px;
+}
+
+.canvas-blue {
+  position: absolute;
+  left: 5%;
+  top: 20px;
+  width: 120px;
+  height: 25px;
+}
```

Comment:
You could style this without the need of absolute positioning. Instead if the height is smaller than the `nav`, then you could create an apply an `items-center` with `align-items: center` property to it and center the element along the cross axis.